crates/interledger-packet/src/packet.rs

```diff
@@ -667,10 +701,71 @@ mod fuzzed {
 
         let e = Packet::try_from(BytesMut::from(&orig[..])).unwrap_err();
         assert_eq!(
+            "Invalid Packet: Reject.ErrorCode was not IA5String",
             &e.to_string(),
-            "Invalid Packet: Reject.ErrorCode was not IA5String"
         );
     }
+
+    #[test]
+    fn fuzzed_0_with_fixed_errorcode() {
```

Comment:
```suggestion
    fn fuzzed_1_with_fixed_errorcode() {
```

following the fuzzed tests naming convention

Reply:
my logic was to reuse the `fuzzed_0` because this was not a new finding per-say, it's was the same the original, just the single offending errorcode byte modified as acceptable (255 => 127).

Perhaps the `1` would had been more in line!